app/docs/md/learn/concepts/lifecycle.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Enhance Lifecycle
+---
+
+<doc-callout level="info" mark="📍">
+
+The journey of an HTTP request through Enhance isn't complicated, but it's helpful to understand how the final HTML document is assembled from the code you author in `app/`.
+
+</doc-callout>
+
+<doc-enhance-diagram class="block mb2">
+</doc-enhance-diagram>
+
+## Routing
+
+When an HTTP request (GET, POST, etc.) arrives, Enhance decides how to route it. First checking for [a matching API function](/docs/learn/starter-project/api) and then [a Page](/docs/learn/starter-project/pages).
+
+```
+my-rad-app.net/login --> app/api/login.mjs --> /app/pages/login.html
```

Comment:
Does `app/api/login.mjs` need to be `/app/api/login.mjs` to match the rest of the paths on this page?

Reply:
Good catch 👍